app/webpack/observations/show/components/annotations.jsx

```diff
@@ -11,9 +11,8 @@ import { termsForTaxon } from "../ducks/controlled_terms";
 class Annotations extends React.Component {
   constructor( props ) {
     super( props );
-    const currentUser = props.config && props.config.currentUser;
     this.state = {
-      open: currentUser ? !currentUser.prefers_hide_obs_show_annotations : true
```

Comment:
I think what we decided on is the open state of the annotations panel should be closed by default, but should remain sticky based on user preferences. For example if I open the tab and reload the page it should stay open. I also think it makes sense to stay sticky for a session even if the user is not logged in.

For example if I'm not logged in and I go to a page that has observations, the section will be closed by default. Then if I open it and user that arrows at the top of the page to go to another observation and that observation also has annotations, the panel should still be viewable. If I leave and come back, it makes sense it would revert to closed. Whereas if I were logged in and came back it would remain open because that preference stuck.

That may complicate things a bit and make it so we need to manage the open state of the annotations panel outside the component and in the larger context of the obs page.